docs/devGuide/githubActions/markbindAction.md

```diff
@@ -0,0 +1,36 @@
+{% set title = "GitHub Actions: markbind-action" %}
+<span id="title" class="d-none">{{ title }}</span>
+
+<frontmatter>
+  title: "{{ title }}"
+  layout: devGuide.md
+  pageNav: default
+</frontmatter>
+
+# {{ title }}
+
+<div class="lead">
+
+A GitHub Action that builds and deploys your MarkBind site. It helps users to streamline their workflow by:
+* Making it easier to deploy without having to write their own actions from scratch
+* Making the workflows consistent and up to date with any changes specific to MarkBind
+
+The source code is at [MarkBind/markbind-action](https://github.com/MarkBind/markbind-action).
+</div>
+
+**Follow these steps to get started on development**
```

Comment:
Was a bit confused on what "development" refers to here - e.g. Are these steps for modifying MarkBind's `markbind-action`, testing `markbind-action` with a local copy, or for creating Github actions for MarkBind in general?

Reply:
Maybe it would be clearer to split these steps up into different sections (I'm confused about what each step should achieve, and where the steps are leading to in general) - e.g. "Setting up", "Testing/modifying the action".